bundle/Form/FieldTypeHandler/DateAndTime.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandler;
+
+use DateTime;
+use Ibexa\Contracts\Core\FieldType\Value;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\FieldDefinition;
+use Ibexa\Core\FieldType\DateAndTime\Value as DTValue;
```

Comment:
Please expand the alias name to `DateTimeValue`.

Reply:
solved in aec51ebe